StRoot/StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -854,10 +854,11 @@ Int_t StPicoDstMaker::MakeWrite() {
 
   mBField = muEvent->magneticField();
 
+#if !defined (__TFG__VERSION__)
   // Get Emc collection
   mEmcCollection = mMuDst->emcCollection();
-  StMuEmcUtil *emcUtil = new StMuEmcUtil();
   if ( !mEmcCollection ) {
+    static StMuEmcUtil *emcUtil = new StMuEmcUtil();
```

Comment:
```suggestion
    static StMuEmcUtil emcUtil;
```
Why are you insisting on using operator 'new' here? Wouldn't an allocation on the stack work as well?